flask_restful/__init__.py

```diff
@@ -316,24 +316,6 @@ def handle_error(self, e):
                 exc_info = None
             current_app.log_exception(exc_info)
 
-        help_on_404 = current_app.config.get("ERROR_404_HELP", True)
-        if code == 404 and help_on_404:
-            rules = dict([(re.sub('(<.*>)', '', rule.rule), rule.rule)
-                          for rule in current_app.url_map.iter_rules()])
-            close_matches = difflib.get_close_matches(request.path, rules.keys())
```

Comment:
`difflib` and `re` are now unused imports, please remove them

Reply:
Fixed